packages/react-core/src/components/ExpandableSection/ExpandableSection.tsx

```diff
@@ -33,12 +34,22 @@ export interface ExpandableSectionProps extends React.HTMLProps<HTMLDivElement>
   isWidthLimited?: boolean;
   /** Flag to indicate if the content is indented */
   isIndented?: boolean;
+  /** Truncates the expandable content to the specified number of lines. */
+  truncateContent?: number;
```

Comment:
Usually a prop with the term 'content' in it is indicating that a ReactNode or _maybe_ a string will be the type of the prop's value. Not sure if something like `truncatedMaxLines` or something else hinting that the type of this prop is a number.